.build/tasks/Common.Functions.ps1

```diff
@@ -209,3 +209,345 @@ function Get-CodeCoverageOutputFileEncoding
 
     return $codeCoverageOutputFileEncoding
 }
+
+function Merge-JaCoCoReports
+{
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.Xml.XmlDocument]
+        $OriginalDocument,
+
+        [Parameter(Mandatory = $true)]
+        [System.Xml.XmlDocument]
+        $MergeDocument
```

Comment:
technically you could also accept just one parameter, an array of file, and merge them together.
That could also be another function calling this one to further separate the concerns. But that'd make the task easier to test.
Speaking of which, a test for the function would be great.

Reply:
So basically you mean create an additional function which does the loading of the files, which in turn then calls this function. The task just calls the new function, correct?

Will try to add some tests